src/App.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Counter from './counter/Counter';
+import Search from './search/Search';
+import GenreToggle from './toggle/GenreToggle';
+
+// 1. React.createElement
+const myFullName = React.createElement('h1', {}, 'Khakim Bakhramov');
+
+// 2. React.Component
+class Greeting extends React.Component {
+    render() {
```

Comment:
It would be nice to use a class component with logic inside, like the Counter component